src/wallet/wallet_rpc_server.cpp

```diff
@@ -4759,6 +4759,29 @@ namespace tools
     return true;
   }
   //------------------------------------------------------------------------------------------------------------------------------
+  bool wallet_rpc_server::on_get_default_fee_priority(const wallet_rpc::COMMAND_RPC_GET_DEFAULT_FEE_PRIORITY::request& req, wallet_rpc::COMMAND_RPC_GET_DEFAULT_FEE_PRIORITY::response& res, epee::json_rpc::error& er, const connection_context *ctx)
+  {
+    if (!m_wallet) return not_open(er);
+    try
+    {
+      int priority = m_wallet->adjust_priority(0);
```

Comment:
```suggestion
      uint32_t priority = m_wallet->adjust_priority(0);
```
For consistency with the rest of the codebase.